.github/workflows/build-test.yaml

```diff
@@ -75,16 +75,11 @@ jobs:
           file: "spicedb/Dockerfile"
           tags: "spicedb:dev,spicedb:updated"
           outputs: "type=docker,dest=/tmp/image.tar"
-      - name: "Tag Test v1.13.0 image"
```

Comment:
Removed now that 1.14 is published (so all images in the update graph can have the same base image now)